src/LinuxTracing/PerfEventReaders.h

```diff
@@ -55,11 +55,11 @@ SchedWakeupWithCallchainPerfEvent ConsumeSchedWakeupWithCallchainPerfEvent(
 SchedSwitchWithCallchainPerfEvent ConsumeSchedSwitchWithCallchainPerfEvent(
```

Comment:
Will we need to return a generic `PerfEvent` also from `ConsumeSchedWakeupWithCallchainPerfEvent` and `ConsumeSchedSwitchWithCallchainPerfEvent`?

Reply:
Yes, I think so. But let's change this when we are touching that code.